implementations/rust/ockam/ockam_command/src/project/static/enroll/after_long_help.txt

```diff
@@ -1,7 +1,17 @@
 ```sh
+# 1) Use hex encoded string enrollment ticket:
+
 # From the admin machine, generate an enrollment ticket
-$ ticket=$(ockam project ticket --attribute component=user)
+$ TICKET=$(ockam project ticket --attribute component=user)
+
+# From the user machine, enroll the local identity to the project using the ticket
+$ ockam project enroll $TICKET --identity control_identity
+
+# 2) Use file enrollment ticket:
+
+# From the admin machine, generate an enrollment ticket, save to a file
+$ ockam project ticket --attribute component=user > ticket.txt
```

Comment:
The convention we've been using is to name enrollment tickets like `$NAME.ticket`

Reply:
done